docker-base-services.yml

```diff
@@ -38,6 +38,8 @@ services:
       - FLYWAY_BASELINE_ON_MIGRATE=true
       - FLYWAY_PLACEHOLDERS_api_db_username=fam_proxy_api
       - FLYWAY_PLACEHOLDERS_api_db_password=test
+      - FLYWAY_PLACEHOLDERS_admin_management_api_db_user=fam_proxy_admin_management_api
```

Comment:
oh btw, do we call it "FLYWAY_PLACEHOLDERS_admin_management_api_db_user" or "FLYWAY_PLACEHOLDERS_admin_management_api_db_username" ? I saw in the flyway.tf file, it's called "admin_management_api_db_username"

Reply:
Good catch, I didn't notice it. I will fix it for just "_user" not name.

Reply:
This is fixed. Thank you!